src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ManageApprenticesController.cs

```diff
@@ -23,29 +21,40 @@ public ManageApprenticesController(ICommitmentsService commitmentsService, ICrea
             _createCsvService = createCsvService;
         }
 
-        public async Task<IActionResult> Index(uint providerId)
+        [Route("manage", Name = RouteNames.ManageApprentices)]
+        public async Task<IActionResult> Index(uint providerId, int pageNumber = 1)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
 
+            var result = await _commitmentsService.GetApprenticeships(providerId, pageNumber, ProviderCommitmentsWebConstants.NumberOfApprenticesPerSearchPage);
+
+            var filterModel = new ManageApprenticesFilterModel
```

Comment:
I'd expect a mapper to do some stuff here and keep the controller light

Reply:
We could use a mapper or just put the model creation into the service. Given its just building up a model (no business logic) and isn't used elsewhere Initially thought it was fairly light already but could create a mapper if required

Reply:
Mapper has been added as requested